From b7b491cc6c907518670f37af6629eb882886d87c Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Tue, 21 Mar 2023 22:23:41 -0300
Subject: [PATCH 3/4] GENIO: arm: dts: mt8195-demo: add psci node

Add missing psci dts node which is required for CONFIG_SYSRESET_PSCI to
work correctly.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 arch/arm/dts/mt8195-demo.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/dts/mt8195-demo.dts b/arch/arm/dts/mt8195-demo.dts
index 5a70d4501b0..8a3287060de 100644
--- a/arch/arm/dts/mt8195-demo.dts
+++ b/arch/arm/dts/mt8195-demo.dts
@@ -25,6 +25,11 @@
 			compatible = "linaro,optee-tz";
 			method = "smc";
 		};
+
+		psci: psci {
+			compatible = "arm,psci-1.0";
+			method = "smc";
+		};
 	};
 
 	reserved-memory {
-- 
2.39.1

